common/beerocks/bcl/source/network/network_utils.cpp

```diff
@@ -84,59 +84,27 @@ const std::string network_utils::MULTICAST_1905_MAC_ADDR("01:80:c2:00:00:13");
 //////////////////////////////////////////////////////////////////////////////
 
 // Converts uint64_t mac address to string format
-std::string network_utils::mac_to_string(const uint64_t mac)
-{
-    uint8_t mac_address[MAC_ADDR_LEN];
-    int8_t i;
-    uint8_t *p = mac_address;
-    for (i = 5; i >= 0; i--) {
-        *p++ = mac >> (CHAR_BIT * i);
-    }
-    return mac_to_string(mac_address);
-}
+std::string network_utils::mac_to_string(const uint64_t mac) { return tlvf::mac_to_string(mac); }
 
 // Converts a mac address in a human-readable format
 std::string network_utils::mac_to_string(const uint8_t *mac_address)
 {
-    std::string mac_addr_string;
-
-    mac_addr_string = string_utils::int_to_hex_string((uint32_t)mac_address[0], 2) + ":" +
-                      string_utils::int_to_hex_string((uint32_t)mac_address[1], 2) + ":" +
-                      string_utils::int_to_hex_string((uint32_t)mac_address[2], 2) + ":" +
-                      string_utils::int_to_hex_string((uint32_t)mac_address[3], 2) + ":" +
-                      string_utils::int_to_hex_string((uint32_t)mac_address[4], 2) + ":" +
-                      string_utils::int_to_hex_string((uint32_t)mac_address[5], 2);
-
-    return mac_addr_string;
+    return tlvf::mac_to_string(mac_address);
 }
 
 std::string network_utils::mac_to_string(const sMacAddr &mac)
 {
-    return mac_to_string((const uint8_t *)mac.oct);
+    return tlvf::mac_to_string((const uint8_t *)mac.oct);
```

Comment:
I think it is better to remove these functions from network utils, and add one more commit that replaces all the calling for these functions to have the `tlvf::` prefix instead of `network_utils::`.

Reply:
You just love these huge changes hu? :)
OK Done, but I really hope it will be merged soon otherwise it will be a nightmare on rebases.